packages/marketplace/src/components/pages/developer-edition-download/download-modal.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from 'react'
+import { Modal, Button } from '@reapit/elements'
+
+interface DeveloperEditionDownloadProps {
+  handleDownload: () => void
+}
+
+export const DeveloperEditionDownload: React.FC<DeveloperEditionDownloadProps> = ({ handleDownload }) => {
```

Comment:
Each component here should have a separate file, so when the logic grows, it doesn't make this file become huge